### PR TITLE
[FIX] html_editor: history: expose canRedo/canUndo

### PR DESCRIPTION
**Pull request by kebeclibre**

### CODE REVIEW

**Comment by kebeclibre** [comment]
@Goaman @FrancoisGe 
Obviously this needs a test, but would that be okay ?
The state of "currentStep" must be done before triggering the onchange, for the getNextIds functions to work properly

File: addons/html_editor/static/src/core/history_plugin.js [hunk 1]
```diff
@@ -431,7 +433,7 @@ export class HistoryPlugin extends Plugin {
         return Math.floor(Math.random() * Math.pow(2, 52)).toString();
     }
 
-    addStep() {
+    addStep(stepState) {
```
Comment (Goaman) [inline addons/html_editor/static/src/core/history_plugin.js]: {stepState} = {}

File: addons/html_editor/static/src/core/history_plugin.js [hunk 2]
```diff
@@ -460,10 +462,19 @@ export class HistoryPlugin extends Plugin {
             selection: {},
             mutations: [],
         });
+        if (stepState) {
```
Comment (Goaman) [inline addons/html_editor/static/src/core/history_plugin.js:480]: add comment

File: addons/html_editor/static/src/core/history_plugin.js (hunk 1 above)
Comment (Goaman) [inline addons/html_editor/static/src/core/history_plugin.js]: add type

**Pull request merged**